### PR TITLE
OSDOCS#5790: Create preparing for update subsection

### PR DESCRIPTION
**Pull request by skopacz1**
[OSDOCS-5790](https://issues.redhat.com/browse/OSDOCS-5790)

Versions: 4.14+

This PR creates a new subsection within the update docs for "Preparing for an update". The subsection contains assemblies with procedures that users might perform before initiating an update. Changes are limited to content reorganization and link updates, so no new content has been created for this PR.

QE is not required.

Previews:

- Preparing to update a cluster
  - [Preparing to update to OpenShift Container Platform 4.13](https://62571--docspreview.netlify.app/openshift-enterprise/latest/updating/preparing_for_updates/updating-cluster-prepare.html)
  - [Preparing to update a cluster with manually maintained credentials](https://62571--docspreview.netlify.app/openshift-enterprise/latest/updating/preparing_for_updates/preparing-manual-creds-update.html)
  - [Preflight validation for Kernel Module Management (KMM) Modules](https://62571--docspreview.netlify.app/openshift-enterprise/latest/updating/preparing_for_updates/kmm-preflight-validation.html)

- OKD: [Preparing to update to OKD 4.13](https://file.rdu.redhat.com/skopacz/OSDOCS-5790/updating/preparing_for_updates/updating-cluster-prepare.html) (VPN required)
- Sample pages from other doc sections with link changes:
  - (Installing) [Manually creating IAM for AWS](https://62571--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/manually-creating-iam.html#:~:text=an%20upgradeable%20state.-,Additional%20resources,-Preparing%20to%20update)
  - (Authentication) [Using manual mode with GCP Workload Identity
](https://62571--docspreview.netlify.app/openshift-enterprise/latest/authentication/managing_cloud_provider_credentials/cco-mode-gcp-workload-identity.html#additional-resources_cco-mode-gcp-workload-identity)

### CODE REVIEW

File: updating/preparing_for_updates/preparing-manual-creds-update.adoc
Comment (skopacz1) [inline updating/preparing_for_updates/preparing-manual-creds-update.adoc:1]: Note to reviewers: This is not actually a new file. For some reason, when I moved this existing file into the new directory, the git diff recorded this as me deleting the old file and creating this new one. The only changes in this file are all the xrefs.

**Review by adellape — COMMENTED** [review]
The link changes looked good to me.

For the modules that are `include`'d in the assemblies that got moved, they could have their "Module included in the following assemblies" lists updated. For example, the `updating-cluster-prepare.adoc` assembly includes the `update-preparing-list.adoc` module, but the list at the top of the module still says:

```
// * updating/updating-cluster-prepare.adoc
```

Which could be updated now to:

```
// * updating/preparing_for_updates/updating-cluster-prepare.adoc
```

Should be simple project-wide find/replaces if you search for the old `// *` locations.

**Comment by skopacz1** [comment]
@adellape Thanks for bringing the module include statements up! The way I planned out this reorg was to try and break things up into discrete chunks of work to facilitate faster overall progress, so I'll be updating the module include statements (all of the ones affected by my update reorg, not just this PR) as part of [OSDOCS-5813](https://issues.redhat.com/browse/OSDOCS-5813).

I'll also add commented warnings to each assembly affected by my reorg, warning others that changes to the assemblies for versions <4.14 will likely need their own PRs, otherwise their cherry picks might fail. That'll be done as part of [OSDOCS-5792](https://issues.redhat.com/browse/OSDOCS-5792)

**Review by skrthomas — COMMENTED** [review]
merge review lgtm

File: updating/preparing_for_updates/preparing-manual-creds-update.adoc
Comment (jeana-redhat) [reply updating/preparing_for_updates/preparing-manual-creds-update.adoc:1]: Thanks for the note re: your changes here. I have changes to this open and this update caused a merge conflict but your note means it's easy for me to fix :nerd_face:

Comment (skopacz1) [reply updating/preparing_for_updates/preparing-manual-creds-update.adoc:1]: Yeah sorry for any inconvenience this caused you! I should have let you known about this change ahead of merging, that's totally my bad

**Pull request merged**